multichain-testing/scripts/fast-usdc.ts

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env -S node --import ts-blank-space/register
+import '@endo/init';
+import { parseArgs } from 'node:util';
+import type { ExecutionContext } from 'ava';
+import { commonSetup } from '../test/support.js';
+import { makeFeedPolicy, oracleMnemonics } from '../test/fast-usdc/config.js';
+import { makeDenomTools } from '../tools/asset-info.js';
+import { makeDoOffer } from '../tools/e2e-tools.js';
+import { AmountMath, type Brand } from '@agoric/ertp';
+import { divideBy } from '@agoric/zoe/src/contractSupport/ratio.js';
+import type { USDCProposalShapes } from '@agoric/fast-usdc/src/pool-share-math.js';
+import type { PoolMetrics } from '@agoric/fast-usdc/src/types.js';
+
+const contractName = 'fastUsdc';
+const contractBuilder =
+  '../packages/builders/scripts/fast-usdc/init-fast-usdc.js';
+
+const mockT = {
```

Comment:
This seems like sort of the opposite of a mock.

```suggestion
const runT = {
```